docs/README.md

```diff
@@ -98,7 +99,7 @@ The Media Communications Mesh (MCM) enables efficient, low-latency media transpo
                 sudo yum install -y libbsd-devel cmake make rdma-core libibverbs-devel librdmacm-devel dracut
                 ```
 
-        - Install the irdma driver and libfabric
+        - Install the irdma driver and libfabric:
```

Comment:
It would be worthy to add here at the end of line:
"(more info on installation instructions)"
with link to the address there: https://github.com/ofiwg/libfabric?tab=readme-ov-file#building-and-installing-libfabric-from-source

Reply:
Please check a comment from @Sakoram above. I am not sure whom to follow at this point.

Reply:
I think we are not disagreeing with each other. I wanted to remove the point about manual installation, Szymon wants to add the link next to the instruction to run the script not to removed point.

Reply:
Thanks! Added a tip note about it. Please check if that suffice.

Reply:
LGTM